test/cql-pytest/test_wasm.py

```diff
@@ -78,6 +78,16 @@ def test_fib(cql, test_keyspace, table1, scylla_with_wasm_only):
         with pytest.raises(InvalidRequest, match="wasm"):
             cql.execute(f"SELECT {test_keyspace}.{fib_name}(p) AS result FROM {table} WHERE p = 997")
 
+# Reads WASM UDF from a file. For a function "foo", the file should be named "foo.wat" and
+# be located in the "test/resource/wasm" directory. Supports renaming the exported function.
+def read_function_from_file(func_name, rename=None):
+    wat_path = os.path.realpath(os.path.join(__file__, f"../../resource/wasm/{func_name}.wat"))
```

Comment:
I don't think this realpath() is needed at all, but I guess it doesn't hurt either.